plugins/antispam/antispam_plugin.cpp

```diff
@@ -1,14 +1,33 @@
-#include <time.h>
+
+/*
+Antispam plugin updated to run on newer versions of Cod4x.
+
+Contributors :
+
+# Maintained/Updated by Kingwolf(Usmania) @ Cod4x 2017
+# Originally Made by TheKelm @ Iceops 2013 for cod4x17a.
+
+
+This plugin is part of the cod4x source code, and is a subset of the cod4x server source code completely,
+hence inheriting any legal licenses that it's parent has.
+*/
+
+ 
 #include "../pinc.h"
+#include <chrono> // for time_t typdef
 
-#define ANTISPAM_MAXMESSAGES 30
+using namespace std;
 
-typedef struct{
+PCL int OnInit();
+#define max 8
```

Comment:
this is a horrible name for a preprocessor definition, please change back to ANTISPAM_MAXMESSAGES